action_files/test_models/src/evaluation.py

```diff
@@ -32,7 +31,7 @@ def evaluate():
 
     eval_tags = {}
     eval_tags['Total'] = tags['Country']
-    eval_tags['Purpose'] = tags['Country/Purpose']
+    # eval_tags['Purpose'] = tags['Country/Purpose']
```

Comment:
Why was this removed?

Reply:
Hierarchy needs to be strictly hierarchical

Reply:
I suggest we remove them instead of commenting them out.

Reply:
Yeah, leaving these here as future-me wants to create additional tests when I'm introducing temporal reconciliation techniques, I probably want to try out strict- and non-strict hierarchical stuff then too.

For now my rationale was to get a reasonable test on every existing method